dask/tests/test_base.py

```diff
@@ -514,3 +514,10 @@ def my_get(dsk, keys):
 
     with dask.set_options(array_optimize=None, get=my_get):
         y.compute()
+
+
+def test_rechunk_zero_dim():
+    da = pytest.importorskip('dask.array')
+
+    x = da.ones((0,10,100), chunks=(0,10,10)).rechunk((0,10,50))
+    assert len(x.compute()) == 0
```

Comment:
This should probably be in dask/array/tests/test_array_core.py

Reply:
Moved to dask/array/tests/test_chunk.py